src/mapd-crossfilter.js

```diff
@@ -1447,7 +1447,7 @@ export function replaceRelative(sqlStr) {
           query +=
             " MOD(" +
             _dataTables[0] +
-            ".rowid * 265445761, 4294967296) < " +
+            ".rowid * 2654435761, 4294967296) < " +
```

Comment:
The number is correct, but we likely need to match the expression in @VrajPandya metis PR here: https://github.com/omnisci/metis/pull/38

That expression reduces the size of the initial multiplication, which will reduce the chance that the multiplication will overflow.

Reply:
thanks @alexbaden . It should be same now.